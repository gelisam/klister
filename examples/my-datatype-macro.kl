(import "datatype-macro.kl")

(my-datatype Nat
  (zero)
  (suc Nat))

(the Nat 42)

(type-case t
  [Nat
   (pure '42)])
