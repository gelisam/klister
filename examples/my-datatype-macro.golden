/app/examples/my-datatype-macro.kl:1:1:
  |
1 | (import "datatype-macro.kl")
  | ^^^^^
unexpected "(impo"
expecting "#lang"
